crates/chia-bls/Cargo.toml

```diff
@@ -19,7 +19,8 @@ arbitrary = ["dep:arbitrary"]
 chia-traits = { workspace = true }
 chia_py_streamable_macro = { workspace = true, optional = true }
 anyhow = { workspace = true }
-sha2 = { workspace = true }
+sha2 = {version = "0.10.8"}
```

Comment:
Use `{ workspace = true }` here. We use [workspace dependencies](https://doc.rust-lang.org/cargo/reference/workspaces.html#the-dependencies-table)

Reply:
changed